build/dt-report-generator-bundle.js

```diff
@@ -11,7 +11,7 @@ const fs = require('fs');
 const distDir = __dirname + '/../dist';
 const outFile = `${distDir}/report-generator.js`;
 const generatorFilename = `./lighthouse-core/report/report-generator.js`;
-browserify(generatorFilename, {standalone: 'ReportGenerator'})
+browserify(generatorFilename, {standalone: 'Lighthouse.ReportGenerator'})
```

Comment:
oh woah it creates the namespace object and everything just based on the `.`? color me impressed browserify :)

Reply:
![image](https://user-images.githubusercontent.com/4071474/54560032-a84f7f80-497e-11e9-9791-ab1d672a7821.png)